CRM/Contribute/Form/ContributionBase.php

```diff
@@ -714,7 +714,7 @@ public function assignToTemplate() {
     $this->assign('currencyID', $this->_params['currencyID'] ?? NULL);
     $this->assign('credit_card_type', $this->_params['credit_card_type'] ?? NULL);
     $this->assign('trxn_id', $this->_params['trxn_id'] ?? NULL);
-    $this->assign('amount_level', $this->order->getAmountLevel());
+    $this->assign('amount_level', str_replace(CRM_Core_DAO::VALUE_SEPARATOR, '', $this->order->getAmountLevel()));
```

Comment:
My only question is it ever something like `^A`Event Fee - 1`^A`Optional Donation - 2`^A`? Is that why this is put together as serialized? In which case this would now look like "Event Fee - 1Optional Donation - 2", with no delimiter in between.

Reply:
@demeritcowboy - that function is also used here to prepare for db write (below) - we might be able to unravel them all to handle an array but that might take a bit of digging - it seems like a space is less evil than an empty string on the replace though?

![image](https://github.com/user-attachments/assets/c2b7379c-2419-4998-959f-268acd34758e)

Reply:
Although as noted in the description it seems to only affect quickconfig, where I don't think you can have that kind of thing. When it's price sets it displays one per line anyway.
So for the immediate purposes this seems ok.

Reply:
Oh yes a space should work.